google-cloud-language/lib/google/cloud/language/document.rb

```diff
@@ -70,10 +70,16 @@ def content?
         ##
         # @private Whether the document source is a Google Cloud Storage URI.
         #
-        def url?
+        def gcs_url?
           @grpc.source == :gcs_content_uri
         end
 
+        def url?
```

Comment:
The previous `url?` method is marked private, and not part of the public API. It does not show up in the documentation, and end users should not be aware of it. Why is this new `url?` method and warning added, which are _not_ longer marked private?

We should be free to make internal implementation changes without this step. Right?